examples/nixnet/ethernet-frame-stream-input-reader.py

```diff
@@ -0,0 +1,179 @@
+  C:\Users\Public\Documents\National Instruments\NI-XNET\Documentation\NI-XNET Manual.chm
+
+Getting Started:
+To run this example, install "NI-XNET Driver" on the server machine:
+  https://www.ni.com/en-in/support/downloads/drivers/download.ni-xnet.html
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC
+Client" wiki page:
+  https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Refer to the NI XNET gRPC Wiki for the latest C Function Reference:
+  https://github.com/ni/grpc-device/wiki/NI-XNET-C-Function-Reference
+
+ Running from command line:
+Server machine's IP address, port number, and interface name can be passed as separate command line
+arguments.
+  > python ethernet-frame-stream-input-reader.py <server_address> <port_number> <interface_name>
+If they are not passed in as command line arguments, then by default the server address will be
+"localhost:31763"
+"""
+
+import sys
+import time
+from datetime import datetime
+
+import grpc
+import nixnet_pb2 as nixnet_types
+import nixnet_pb2_grpc as grpc_nixnet
+
+
+CHOOSE_MONITOR_OR_ENDPOINT_TEXT = (
+    "\nPress 'm' followed by enter to configure the input stream session to use the monitor path"
+    " to monitor all network traffic else just the enter key to use the endpoint path which filters"
+    " traffic based on VLAN ID and priority. :"
+)
+FRAME_CHECK_SEQUENCE_SIZE = 4
+MAX_ENET_FRAME_SIZE = 1518
+NUM_OF_FRAMES = 1
+MAX_PAYLOAD_PER_FRAME = MAX_ENET_FRAME_SIZE + FRAME_CHECK_SEQUENCE_SIZE
+
+# Parameters
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+INTERFACE = "ENET1"
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    SERVER_ADDRESS = sys.argv[1]
+if len(sys.argv) >= 3:
+    SERVER_PORT = sys.argv[2]
+if len(sys.argv) >= 4:
+    INTERFACE = sys.argv[3]
+
+
+def check_for_error(status):
+    """Raise an exception if the status indicates an error."""
+    if status != 0:
+        error_message_response = client.StatusToString(
+            nixnet_types.StatusToStringRequest(status_id=status)
+        )
+        raise Exception(error_message_response.status_description)
+
+
+def print_timestamp(timestamp):
+    """Print a timestamp to console."""
+    local_dt = datetime.fromtimestamp(timestamp / 1e9)
+    print(local_dt.strftime("%m/%d/%Y %H:%M:%S.%f")[:-3], end=" ")
+
+
+# Declare all variables for the function
+count = 0
+session = None
+
+# Receive filter only applies for endpoint mode.
+flag = 3
+vid = 2
+priority = 3
+mac = "AA:BB:CC:DD:EE:FF"  # MAC Address filter is not enabled. Use random string.
+
+# Create the communication channel for the remote host and create connections to the NI-XNET and
+# session services.
+channel = grpc.insecure_channel(f"{SERVER_ADDRESS}:{SERVER_PORT}")
+client = grpc_nixnet.NiXnetStub(channel)
+
+print(CHOOSE_MONITOR_OR_ENDPOINT_TEXT)
+if sys.stdin.read(1) == "m":
+    INTERFACE = "ENET1/monitor"
```

Comment:
This should be `INTERFACE += "/monitor"`, otherwise we are overwriting interface named passed through command line parameter.

Reply:
Done.